opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -447,7 +442,6 @@ message IntHistogramDataPoint {
   // distribution of values in the histogram is unknown and only the total count and sum are known.
 
   // explicit_bounds is the only supported bucket option currently.
-  // TODO: Add more bucket options.
```

Comment:
This is removed.

Reply:
yeah, other histogram types are tracked as bugs/oteps now.

Reply:
I see what you meant, didn't notice the merge conflict :)